records_mover/records/targets/table/move_from_fileobjs_source.py

```diff
@@ -53,8 +53,8 @@ def reset_before_reload(self) -> None:
         self.fileobj.seek(0)
 
     def move(self) -> MoveResult:
-        with self.tbl.db_engine.begin() as db:
-            driver = self.tbl.db_driver(db)
+        with self.tbl.db_engine.begin() as db_conn:
```

Comment:
same note here as my previous note on using `db_engine.begin()` vs first opening a connection and using `conn.begin()`

Reply:
on it

Reply:
Updated these across the project (there were several)